src/lib.rs

```diff
@@ -1,8 +1,73 @@
-#[cfg(test)]
-mod tests {
-    #[test]
-    fn it_works() {
-        let result = 2 + 2;
-        assert_eq!(result, 4);
-    }
+// Copyright 2019-2022 Parity Technologies (UK) Ltd.
+// This file is part of subxt.
```

Comment:
```suggestion
// This file is part of scale-value.
```

Reply:
Ooh good spot; need to go around and change these copyright headers now :)